src/components/App.vue

```diff
@@ -3,7 +3,13 @@
 </template>
 
 <style>
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
 body {
-  background-color: yellow;
+  font-family: 'Fira Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace;
```

Comment:
@jjlumagbas This might be interesting about font stacks: https://css-tricks.com/snippets/css/font-stacks/

Reply:
I actually took this font stack from: https://www.client9.com/css-system-font-stack-monospace-v2/

Reply:
Yes this is okay, no need to change. Was just sharing haha 🎉